index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <div>
+            <nav class="navbar">
+                <img class="nav-image" src="images/image1.png" alt="image1">
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </div>
+        <div class="section">
+            <h1>Rudy<br>Menekam</h1>
+            <p> Hello! I am a software developer and designer! I can help you build a product, feature or website. Taka a look at my works. If you like what you see and have a project you need coded or designed, don’t hesitate and contact me.</p>
+            <a href="#" class="button">Start collaboration</a>
+        </div>
+    </div>
+    <div class="section-image">
+    </div>
+    </div>
```

Comment:
- [ ] The container of the section should be  a `section` tag. and this will represent a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. kindly take care of this :wink: